packages/flutter/test/painting/image_stream_test.dart

```diff
@@ -811,44 +811,31 @@ void main() {
     expect(oneFrameCodec.numFramesAsked, 1);
   });  // https://github.com/flutter/flutter/issues/82532
 
-  // TODO(amirh): enable this once WidgetTester supports flushTimers.
```

Comment:
https://github.com/flutter/flutter/issues/30344 and its duplicate https://github.com/flutter/flutter/issues/24166 are both closed. Synced with @dnfield offline to see if this test actually works

Reply:
Test does not work, I'm not even really sure what it was supposed to test, and multiple things it referred to have been refactored. We've made it this far without this test so we're probably ok.

Reply:
Got rot. Thanks for checking!